deploy/examples/common-second-cluster.yaml

```diff
@@ -288,3 +288,57 @@ kind: ServiceAccount
 metadata:
   name: rook-ceph-rgw
   namespace: rook-ceph-secondary # namespace:cluster
+---
+# Allow the ceph mgr to access cluster-wide resources necessary for the mgr modules
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-mgr-secondary-cluster
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-mgr-secondary-cluster
+subjects:
+  - kind: ServiceAccount
+    name: rook-ceph-mgr
+    namespace: rook-ceph # namespace:cluster
+---
+# Aspects of ceph-mgr that require cluster-wide access
+kind: ClusterRole
```

Comment:
Since this clusterrole is already defined in common.yaml, we should actually be able to assume it already is defined. ClusterRoles only need to be defined once, if there is no difference needed in the different namespace. No need for defining this ClusterRole here.